InvenTree/build/templates/build/detail.html

```diff
@@ -106,7 +106,7 @@ <h4>{% trans "Build Details" %}</h4>
                 <tr>
                     <td><span class='fas fa-link'></span></td>
                     <td>{% trans "External Link" %}</td>
-                    <td><a href="{{ build.link }}">{{ build.link }}</a>{% include "clip.html"%}</td>
+                    <td><a href="{{ build.link }}" rel="noreferrer" target="_blank">{{ build.link }}</a>{% include "clip.html"%}</td>
```

Comment:
I think it would be worth breaking this out into a template e.g.

```html
{% include "clip_link.html" with link=build.link %}
```

And then the template includes the clipboard functionality also.

Reply:
Yeah, I see what you mean.
Small braindump:
A general rule of thumb for clip_link.html will then be that it serves the URL as the string of the anchor tag. 
In addition, it'll be targeted at links to be opened in new tabs, mainly external ones. 
If a more generic approach would be ideal, more parameter for string value and external toggle could be added to the template. 
LMK your thoughts, and I'll update the PR to use a template